File: src/ObjectManager/Object.Tes/ConvertUtils.cs
using UnityEngine;

namespace OA.Tes
{
    public static class ConvertUtils
    {
        const int yardInUnits = 64;
        const float meterInYards = 1.09361f;
        public const float MeterInUnits = meterInYards * yardInUnits;

        const int exteriorCellSideLengthInUnits = 128 * yardInUnits;
        public const float ExteriorCellSideLengthInMeters = (float)exteriorCellSideLengthInUnits / MeterInUnits;

        public static Quaternion RotationMatrixToQuaternion(Matrix4x4 matrix)
        {
            return Quaternion.LookRotation(matrix.GetColumn(2), matrix.GetColumn(1));
        }
    }
}

File: src.mac/ObjectManager/ObjectManager/Object.Tes/ConvertUtils.swift
//
//  ConvertUtils.swift
//  ObjectManager
//
//  Created by Sky Morey on 5/28/18.
//  Copyright © 2018 Sky Morey. All rights reserved.
//

import simd

public class ConvertUtils {
    static let yardInUnits = 64
    static let meterInYards: Float = 1.09361
    public static let meterInUnits: Float = meterInYards * Float(yardInUnits)
    static let exteriorCellSideLengthInUnits = 128 * yardInUnits
    public static let exteriorCellSideLengthInMeters = Float(exteriorCellSideLengthInUnits) / meterInUnits
}
